### ISSUE TITLE
Hi, i want to ask how to check the accuracy of training？

### ISSUE THREAD

**Issue by anglgn** [artitw/text2class#12]
I see the code for checking accuracy, but I don't know how to use it.

**Comment by artitw** [comment]
1. Which part fo the code are you referring to?
2. Are you looking for the final accuracy?

**Comment by anglgn** [comment]
Yes, i just want the final accuracy.

**Comment by anglgn** [comment]
Yes, i just want to get the final accuracy.

**Comment by artitw** [comment]
With the dataset you want to calculate accuracy on, run it through the model to get predictions, then you can compare the predictions with the ground truth to calculate accuracy. If you really need a library, take a look at https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html
